install.sh

```diff
@@ -461,6 +474,10 @@ if [[ "${#mkdirs[@]}" -gt 0 ]]; then
   printf "%s\n" "${mkdirs[@]}"
 fi
 
+if [[ -z "${HOMEBREW_MULTI_USER-}" ]]; then
+  echo "Set HOMEBREW_MULTI_USER env var to allow ${tty_underline}${GROUP}${tty_reset} group users to install packages"
+fi
```

Comment:
Could it be the default behavior instead, to avoid both the extra complexity in implementation and the extra step from the users? 🤔

Reply:
Eventually: yes. Now: no.